MapboxDirections/MBRouteOptions.swift

```diff
@@ -211,6 +211,13 @@ open class RouteOptions: NSObject {
      */
     open var routeShapeResolution = RouteShapeResolution.low
     
+    /**
+     AttributeOptions for the route. Any combination of `.AttributeOptions` can be specified. 
+     
```

Comment:
Note that this option requires the `routeShapeResolution` property to be set to `.full`. (The default is `.low`.)